src/xcode/ENA/ENA/Source/Scenes/ErrorReporting/ErrorReportHistory/ErrorReportHistoryViewModel.swift

```diff
@@ -0,0 +1,62 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+final class ErrorReportHistoryViewModel {
+
+	// MARK: - Init
+
+	init() {
+		// TO DO: should be replaced by Ids
```

Comment:
remove ToDo

Reply:
It's on purpose.

Reply:
We keep this TODO for now. Otherwise we loose track on the open stuff.